Microsoft.Azure.Cosmos/src/Resource/Container/Container.cs

```diff
@@ -1032,6 +1032,7 @@ public abstract class Container
         /// <param name="allowSynchronousQueryExecution">(Optional)the option which allows the query to be executed synchronously via IOrderedQueryable.</param>
         /// <param name="continuationToken">(Optional) The continuation token in the Azure Cosmos DB service.</param>
         /// <param name="requestOptions">(Optional) The options for the item query request.</param>
+        /// <param name="cosmosPropertyNamingPolicy">(Optional) To decide whether property names will be read as camelcase in LINQ. This flag overrides the CosmosSerializationOptions.</param>
```

Comment:
This is not just the camel case, it is exposing all other options, like IgnoreNull.

```suggestion
        /// <param name="cosmosPropertyNamingPolicy">(Optional) Customizes the serialization options to be used to resolve properties for this LINQ query. This flag overrides the CosmosSerializationOptions used on the CosmosClient initiaization.</param>
```